src/app/components/account/account-list-item-layout.tsx

```diff
@@ -19,6 +22,12 @@ interface AccountListItemLayoutProps extends StackProps {
   balanceLabel: React.ReactNode;
   onSelectAccount(): void;
 }
+
```

Comment:
Adding this so we don't show a `.` when in ledger mode and only one address type

![Screenshot 2023-12-15 at 09 12 24](https://github.com/leather-wallet/extension/assets/2938440/85eac4f7-aa58-4652-9a67-f09e0a81cb3e)

I'm not sure if we don't want to export the `WalletType` enum or if there is a preferred way to do this? 

I wasn't sure how to do it as cleanly with: `whenWallet(`

Reply:
Reverted this as theres a bit [more to it](https://github.com/leather-wallet/extension/issues/4596)